README.md

```diff
@@ -1,11 +1,11 @@
 # OCS Example Project
 
-
 ## A pre-populated demo of the OCS services
 
-This example project provides a docker-compose file which launches an [Observation Portal](https://github.com/observatorycontrolsystem/observation-portal), a [Configuration Database](https://github.com/observatorycontrolsystem/configdb), a [Downtime Database](https://github.com/observatorycontrolsystem/downtime), and an [Adaptive Scheduler](https://github.com/observatorycontrolsystem/adaptive_scheduler) all connected together with pre-populated sample data.
+This example project provides a docker-compose file which launches an [Observation Portal](https://github.com/observatorycontrolsystem/observation-portal), a [Configuration Database](https://github.com/observatorycontrolsystem/configdb), a [Downtime Database](https://github.com/observatorycontrolsystem/downtime), an [Adaptive Scheduler](https://github.com/observatorycontrolsystem/adaptive_scheduler), and a [web interface](https://github.com/observatorycontrolsystem/ocs-example-frontend) all connected together with pre-populated sample data.
```

Comment:
I would put the web interface part directly after the Observation Portal and make it clear that it is the web interface for the Observation Portal